### ISSUE TITLE
No response from model

### ISSUE THREAD

**Issue by pavlosidelov** [weaviate/Verba#195]
## Description
Verba launched properly. OpenAI, Google API Key, OLLAMA are installed and available properly. 2 PDFs are imported properly with OLLAMA. On all requests the same error appears.
LOG:
Received query: What is Finance
⚠ Query failed: 'NoneType' object is not iterable
In UI
Something went wrong: 'NoneType' object is not iterable

**Comment by thomashacker** [comment]
Thanks for the issue, we'll have a look. Can you provide more information about how you installed Verba?

**Comment by soniverse-jm** [comment]
Hello, today I've installed Verba and Weaviate inside a Docker environment. Before I installed Ollama with Ollama-WebGUI and llama3 model.

Now I like to connect Verba with Ollama and the llama3 model. I have the same error like @pavlosidelov . I can start the Verba and the Weaviate Docker container successfully. Both are running and the health checks are fine.

With Safari or Firefox I tried to connect to the webgui of Verba. This works finde, too. When I now insert some text, let say "hello", the log of the container shows:
`✔ Received query: hello
⚠ Query failed: 'NoneType' object is not iterable`

The frontend shows "Retrieving chunks" in an infinite loop...

I used the docker-compose.yml file of your project. I tired version 1.0.0 and version 1.0.2. Both have the same errors on my installation. A difference to @pavlosidelov is that I only configured Ollama (WEAVIATE_URL_VERBA, OLLAMA_URL, OLLAMA_MODEL) and not OpenAPI and Google. Could this be a problem? 

I hope you will find a suitable solution soon.

Thank you very much so far.
